h2o-core/src/test/java/water/H2OTest.java

```diff
@@ -33,5 +35,26 @@ private void checkObjectId(String id, int seq) {
     assertTrue(id.startsWith("test-desc_test-type"));
     assertTrue(id.endsWith("_" + seq));
   }
+
+  @Test
+  public void testPropertiesMatching() throws Exception {
+    final Field ignored_properties = H2O.class.getDeclaredField("IGNORED_PROPERTIES");
```

Comment:
you could have just made it package-private and not have to use reflection

Reply:
Why not ? Done !